arangod/Scheduler/Scheduler.cpp

```diff
@@ -370,16 +446,26 @@ bool Scheduler::pushToFifo(size_t fifo, std::function<void()> const& callback) {
   return true;
 }
 
-bool Scheduler::popFifo(size_t fifo) {
-  int64_t p = fifo - 1;
-  TRI_ASSERT(0 <= p && p < NUMBER_FIFOS);
+bool Scheduler::popFifo(int64_t fifo) {
+  TRI_ASSERT(0 <= fifo && fifo < NUMBER_FIFOS);
```

Comment:
if `fifo` is supposed to be >= 0, why has the function parameter type been changed from `size_t` to `int64_t`?

Reply:
it was more convenient someplace else